core/src/autogluon/core/searcher/bayesopt/gpautograd/warping.py

```diff
@@ -40,45 +39,39 @@ def _rescale(self, x):
         """
         We linearly rescale the entries of x into [NUMERICAL_JITTER, 1-NUMERICAL_JITTER]
```

Comment:
just want to make sure:  this jitter-rescaling does not prevent HPO from considering the min/max value in a hyperparameter-range correct? Eg. if dropout is searched over [0,0.5], then 0 could be returned during HPO.

Reply:
No, this _rescale maps x in [lower, upper] to y in [eps, 1-eps].